packages/data-viz/jest.setup.ts

```diff
@@ -0,0 +1,2 @@
+import "@testing-library/jest-dom";
+import "../../intersectionObserverMock";
```

Comment:
Looks like data-viz doesn't have any tests, so we could remove jest and its setup from it if we wanted to simplify things a bit.

Reply:
I think @masoudmanson will start adding tests in December for data-viz Heatmap, so let's keep it 💪 Thank you!